lua/notebook-navigator/repls.lua

```diff
@@ -35,6 +37,21 @@ repls.toggleterm = function(start_line, end_line, repl_args)
   vim.api.nvim_win_set_cursor(current_window, { cursor_line, cursor_col })
 end
 
+-- molten
+---@diagnostic disable-next-line: unused-local
+repls.molten = function(start_line, end_line, repl_args, cell_marker)
+  local line_count = vim.api.nvim_buf_line_count(0)
+
+  if line_count < (end_line + 1) then
+    vim.api.nvim_buf_set_lines(0, end_line + 1, end_line + 1, false, { cell_marker, "" })
+  end
+
+  local ok, _ = pcall(vim.fn.MoltenEvaluateRange, start_line, end_line + 1)
```

Comment:
With the latest release of molten (1.3.0) running a command without a kernel initialized will prompt the user for a kernel, and it will run the code after the kernel is ready.

Reply:
@benlubas Nice! Thanks for the heads up. Will modify the code accordingly soon.